pkg/registries/docker/docker_test.go

```diff
@@ -50,8 +54,45 @@ func TestRegistryMatch(t *testing.T) {
 				cfg:                  &Config{DisableRepoList: tc.disableRepoList},
 			}
 
+			// Prevent lazy load from attempting to change repo list under test.
+			r.repoListOnce.Do(func() {})
+
 			match := r.Match(img)
 			assert.Equal(t, tc.expected, match)
 		})
 	}
 }
+
+func TestLazyLoadRepoList(t *testing.T) {
+	numRepoListCalls := 0
```

Comment:
nit: usually this repo does `var repoListCalls int`

Reply:
https://github.com/stackrox/stackrox/pull/11186/commits/8b8ad991e290224d423a05ab23575fa59c6f4df4